crypto/jwsutil/envelope.go

```diff
@@ -1,53 +0,0 @@
-package jwsutil
```

Comment:
May I know why the `jwsutil` package is removed?

Reply:
Note that this package is implemented according to RFC 7515 and it is not tied to the notation specs.

Reply:
The only internal consumer was `signature/jws`, and after some simplifications `jwsutil` was no longer imported anywhere.

I've you think it make sense to keep it for external consumers, I can revert the removal.

Reply:
Although it is internal, that package is well designed / implemented / decoupled and can be used later.

Reply:
The I'll add it again.